src/screens/FeedScreen/index.js

```diff
@@ -63,6 +64,8 @@ const FeedScreen = (props) => {
     setUpVote
   } = useCoreFeed();
 
+  const {topics} = useDiscovery();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While the `useDiscovery` hook is being used to fetch topics, it's not clear how error handling is done in case fetching fails. It would be good to have some error handling mechanism here.
```diff
- const {topics} = useDiscovery();
+ const {topics, error} = useDiscovery();
+ if (error) {
+   // handle error
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->